src/ch/ehi/umleditor/application/DomainDefDialog.java

```diff
@@ -631,22 +646,28 @@ private javax.swing.JPanel getPnlDetail() {
 			constraintsChxAbstract.insets = new java.awt.Insets(20, 20, 2, 417);
 			getPnlDetail().add(getChxAbstract(), constraintsChxAbstract);
 
-			java.awt.GridBagConstraints constraintsChxMandatory = new java.awt.GridBagConstraints();
-			constraintsChxMandatory.gridx = 1; constraintsChxMandatory.gridy = 3;
-			constraintsChxMandatory.anchor = java.awt.GridBagConstraints.NORTHWEST;
-			constraintsChxMandatory.ipadx = 42;
-			constraintsChxMandatory.insets = new java.awt.Insets(2, 20, 16, 417);
-			getPnlDetail().add(getChxMandatory(), constraintsChxMandatory);
-
 			java.awt.GridBagConstraints constraintsChxFinal = new java.awt.GridBagConstraints();
 			constraintsChxFinal.gridx = 1; constraintsChxFinal.gridy = 2;
 			constraintsChxFinal.anchor = java.awt.GridBagConstraints.NORTHWEST;
 			constraintsChxFinal.ipadx = 83;
 			constraintsChxFinal.insets = new java.awt.Insets(3, 20, 1, 417);
 			getPnlDetail().add(getChxFinal(), constraintsChxFinal);
 
+			java.awt.GridBagConstraints constraintsChxMandatory = new java.awt.GridBagConstraints();
+			constraintsChxMandatory.gridx = 1; constraintsChxMandatory.gridy = 3;
+			constraintsChxMandatory.anchor = java.awt.GridBagConstraints.NORTHWEST;
+			constraintsChxMandatory.ipadx = 42;
+			constraintsChxMandatory.insets = new java.awt.Insets(3, 20, 1, 417);
+			getPnlDetail().add(getChxMandatory(), constraintsChxMandatory);
+
```

Comment:
REQ: Zeilen nicht verschieben, sofern keine Änderungen daran notwendig sind.

Reply:
Die checkboxes werden jetzt in der Reihenfolge initialisiert, wie sie auch im UI erscheinen. Siehe _gridy_ variable des GridBagConstraints.

Reply:
Dann finde ich den Change akzeptabel. Eine gute Begründung.